src/docfx/publish/PublishModel.cs

```diff
@@ -18,6 +18,6 @@ internal class PublishModel
 
         public PublishItem[] Files { get; set; }
 
-        public IReadOnlyDictionary<string, IReadOnlyList<string>> MonikerGroups { get; set; } = new Dictionary<string, IReadOnlyList<string>>();
+        public IReadOnlyDictionary<string, MonikerGroup> MonikerGroups { get; set; } = new Dictionary<string, MonikerGroup>();
```

Comment:
If `moniker_range` is already in `PublishItem`, why duplicating it here?

Reply:
add a `[JsonIgnore]` attribute to PublishItem `ConfigMonikerRange` property